spec/sms_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "sms"
+
+describe SMS do
+  subject(:sms) { described_class.new(client: client) }
+
+  let(:client) { double(:client) }
+
+  it "delivers an SMS with the estimated time" do
+    sms.deliver
+  end
+end
```

Comment:
https://github.com/makersacademy/course/blob/main/pills/levels_of_stubbing.md

This article would probably help a lot for Stubbing, I didn't get a chance to look at it though!